ExampleGallery/UAP/CameraEffectExample.xaml.cs

```diff
@@ -27,12 +28,15 @@ public sealed partial class CameraEffectExample : UserControl, ICustomThumbnailS
         const string noEffect = "No effect";
         const string displacementEffect = "Displacement effect";
         const string rotatingTilesEffect = "Rotating tiles effect";
+        const string variableEffect = "Variable effect";
```

Comment:
Agreed, I will change the effect string name as well as the class itself to be the name of the effect rather than the example idea.